knowledge-base/inputs-disable-copy-paste.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Disabling Copy and Paste in Telerik Blazor Inputs
+description: A guide on preventing users from copying and pasting text into the Telerik Blazor Inputs in a Blazor application.
```

Comment:
The goal is to use the infinitive + "how to"

```suggestion
description: A guide that shows how to prevent users from copying and pasting text into the Telerik Blazor Inputs in a Blazor application.
```